tests/arq/README.md

```diff
@@ -0,0 +1,44 @@
+# Arquillian Tests for JBoss EAP
+
+This project contains OpenShift v3 tests for the JBoss EAP image and
```

Comment:
and .... ?

Reply:
... and I was going to fast for my own good ;)